### PR TITLE
feat: zset cmd zremrangebyscore

### PR DESCRIPTION
**Pull request by callme-taota**
https://github.com/OpenAtomFoundation/pikiwidb/issues/30
ready for review
image

### CODE REVIEW

**Comment by Mixficsol** [comment]
@callme-taota zadd is merged.
https://github.com/OpenAtomFoundation/pikiwidb/pull/143

File: src/cmd_zset.cc [hunk 1]
```diff
@@ -217,4 +217,32 @@ void ZRangebyscoreCmd::DoCmd(PClient* client) {
...
+ZRemRangeByScoreCmd::ZRemRangeByScoreCmd(const std::string& name, int16_t arity)
+    : BaseCmd(name, arity, kCmdFlagsWrite, kAclCategoryWrite | kAclCategorySortedSet) {}
+
+bool ZRemRangeByScoreCmd::DoInitial(PClient* client) {
+  client->SetKey(client->argv_[1]);
+  return true;
+}
+
+void ZRemRangeByScoreCmd::DoCmd(PClient* client) {
+  double min_score = 0, max_score = 0;
```
Comment (LuckyTom1024) [inline src/cmd_zset.cc]: 一行不要定义多个变量吧，下面的也一样处理

File: src/cmd_zset.cc [hunk 2]
```diff
@@ -217,4 +217,32 @@ void ZRangebyscoreCmd::DoCmd(PClient* client) {
...
+
+  int32_t s_ret = 0;
+  storage::Status s = PSTORE.GetBackend(client->GetCurrentDB())
+                          ->ZRemrangebyscore(client->Key(), min_score, max_score, left_close, right_close, &s_ret);
+  if (s.ok()) {
+    client->AppendInteger(s_ret);
+  } else {
+    client->SetRes(CmdRes::kErrOther, s.ToString());
+  }
+  return;
```
Comment (LuckyTom1024) [inline src/cmd_zset.cc]: return去掉

**Comment by callme-taota** [comment]
@panlei-coder @dingxiaoshuai123 review plz

**Review by AlexStocks — APPROVED** [review]

**Review by Mixficsol — APPROVED** [review]

**Pull request merged**